packages/manager/src/auth/createPrismicAuthManager.ts

```diff
@@ -0,0 +1,11 @@
+import { PrismicAuthManager } from "./PrismicAuthManager";
+
+type CreatePrismicAuthManager = ConstructorParameters<
```

Comment:
Yes, but this method allows us to change how `PrismicAuthManager` is written without affecting `createPrismicAuthManager()`.

For example, if we decided to remove the `PrismicAuthManagerConstructorArgs` type and inline `PrismicAuthManager`'s arguments, we don't need to touch this file since it reads `ConstructorParameters<...>`; the types would automatically be updated.